.claude/skills/update-from-design/SKILL.md

```
@@ -0,0 +1,109 @@
+runuser -u reader -- env HOME=/home/reader git -C /home/reader/somnia fetch origin <branch>
+runuser -u reader -- env HOME=/home/reader git -C /home/reader/somnia reset --hard origin/<branch>
+systemctl --user -M reader@ restart somnia-serve
+'
+```
+
+**Verify over the tailnet URL, not localhost.** He reads it at
+`https://srv1701493.tail2221d6.ts.net:8443`; `127.0.0.1:8721` is a different path
+and checking it once produced a confident "deployed and verified" while he was
+looking at a stale tab.
+
+```bash
```

markdownlint-cli2 (0.23.2)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)